### ISSUE TITLE
gPodder doesn't work with NVDA

### ISSUE THREAD

**Issue by patricus3** [gpodder/gpodder#1857]
hello.
when I installed gPodder, it didn't work with NVDA, tabbing, shift tab, nothing, arrows, nothing.
I think you're using an inaccessible GUI
could you maybe switch to something that, actually works for NVDA?

**Comment by computerfreak105** [comment]
Just to let you know, I fully support your request. The grafical user interface isn't really accessible anymore for people using a screen reader. You can do some limited things with JAWS or VoiceOver on mac, but actually navigating all your subscribtion or even add some isn't really possible, or only if you have JAWS and press Insert + Space, O and W for OCR recognition before every mouse click you want to generate. Not a fun experience and it's not warranted that it works every time.

I adjust the settings.opml file to my liking. And I add subscribtions by preparing an .OPML file which I import through the CLI. I use the GUI to just let in run and download the episodes (or in my case the youtube channels) automatically.

**Comment by elelay** [comment]
Hi @computerfreak105 @patricus3. We are sadly at the same state as in #941, ie we still depend on the graphical toolkit for accessibility and I've not heard off any cross-platform progress on Gtk accessibility.
gPodder should work reasonably well with ORCA on Linux, other platforms are not supported.
Also thank you @computerfreak105 to point to the `gpo` command-line interface tool, which is meant to let one do much from the terminal, with hopefully good accessibility from screen readers on all platforms.
Please let us know if improvements could be made in `gpo`, since it is directly under our control.
I'm closing the issue in favor of #941
